src/main/scala/scala/swing/package.scala

```diff
@@ -14,6 +22,8 @@ package object swing {
   type Image      = java.awt.Image
   type Font       = java.awt.Font
 
+  type Seq[+A]    = scala.collection.Seq[A] // because scala.Seq differs between Scala 2.12 and 2.13
+
```

Comment:
I dont understand what this line is for..

Reply:
It can be removed and two spots fully qualified instead: `Container.Wrapper.contents` and `ListView.selection.items`.

Reply:
On the other hand, I found it quite handy in other libs to use `swing.Seq`. But I guess that's a bit idiosyncratic. So I remove it?

Reply:
On the other hand, I found it quite handy in other libs to use swing.Seq. But I guess that's a bit idiosyncratic. So I remove it?

Reply:
@som-snytt `Seq` is used in more than those three places: ComboBox, Container, FileChooser, ListView, Menu, RichWindow, RootPanel, ScrollPane, SplitPane, Table. I'd say it's better and safer to have a type alias in the swing package object.

Reply:
Im fine with that if you want @Sciss

Reply:
I'd remove it, I think it can be confusing. If you were writing the library now, for 2.13, would you create such an alias?